conf/test.config

```diff
@@ -1,28 +1,32 @@
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Nextflow config file for running minimal tests
+    Nextflow config file for running full-size tests
```

Comment:
A lot of leftovers from making this file from the full test one

Reply:
`test.config` looks essentially identical to `test_full.config`. Is there any difference ?

Reply:
These have now been updated. 
test.config = s3 test data
test_full.config = For testing on lustre on farm

Reply:
I couldn't get s3 locations to a folder to work. So I'm reverting that for now.

Reply:
Did you try something like this instead of `GrabFiles` ?

```
ch_grabbed_reads_path   = reads_path.flatMap { Channel.fromPath(it + "/*.fasta.gz").collect() }
```
(inspired from https://nextflow.slack.com/archives/C02T98A23U7/p1715155309566029 and https://nextflow.slack.com/archives/C02T98A23U7/p1708017808290399 )

Reply:
I've just re-tried it to see why it wouldn't happen and i got:
```
ERROR ~ Unexpected error [UnsupportedOperationException]
```

and in the nextflow.log
```
Jul-04 11:29:53.056 [PathVisitor-1] DEBUG nextflow.cloud.aws.nio.S3Client - Setting S3 glacierRetrievalTier=null
Jul-04 11:29:53.073 [Actor Thread 13] ERROR nextflow.extension.DataflowHelper - @unknown
java.lang.UnsupportedOperationException: null
```

So no file found?

Can we push this to a v1.1.0 for now. I've got requests to make the pipeline available.

Reply:
Oki. I've created #22